service/arm_templates/availability_sets_setup.json

```diff
@@ -35,8 +35,11 @@
       "name": "[variables('availabilitySetName')]",
       "location": "[resourceGroup().location]",
       "properties": {
-        "PlatformFaultDomainCount": "[parameters('faultDomainCount')]",
-        "PlatformUpdateDomainCount": "[parameters('updateDomainCount')]"
+        "platformFaultDomainCount": "[parameters('faultDomainCount')]",
+        "platformUpdateDomainCount": "[parameters('updateDomainCount')]"
+      },
+      "sku": {
+        "name": "aligned"
```

Comment:
What is this?

Reply:
This sets the availability set as managed instead of classic. A managed availability set is required for managed disks.

https://docs.microsoft.com/en-us/rest/api/compute/availabilitysets/availabilitysets-create#request